.github/workflows/ci.yml

```diff
@@ -63,7 +63,7 @@ jobs:
   fedora34:
```

Comment:
Please rename, either to `fedora36` or just `fedora`.

Reply:
Ugh, of course. Done.